GUI/Properties/Resources.resx

```diff
@@ -333,4 +336,30 @@ Can't update automatically, because ckan.exe is read-only or we are not allowed
 Do you want to allow CKAN to do this? If you click no you won't see this message again.</value></data>
   <data name="UtilCopyLink" xml:space="preserve"><value>&amp;Copy link address</value></data>
   <data name="StatusInstanceLabelText" xml:space="preserve"><value>Game instance: {0} (KSP {1})</value></data>
+  <data name="ModuleLabelListFavourites" xml:space="preserve"><value>Favourites</value></data>
+  <data name="ModuleLabelListHidden" xml:space="preserve"><value>Hidden</value></data>
+  <data name="ModuleLabelListGlobal" xml:space="preserve"><value>Global</value></data>
+  <data name="EditLabelsDialogConfirmDelete" xml:space="preserve"><value>Are you sure you want to delete {0}? This can't be undone!</value></data>
+  <data name="EditLabelsDialogSavePrompt" xml:space="preserve"><value>Save changes?</value></data>
+  <data name="EditLabelsDialogNoRecord" xml:space="preserve"><value>No record being edited!</value></data>
+  <data name="EditLabelsDialogNameRequired" xml:space="preserve"><value>Name is required!</value></data>
+  <data name="EditLabelsDialogAlreadyExists" xml:space="preserve"><value>{0} already exists in {1}!</value></data>
+  <data name="EditLabelsDialogDelete" xml:space="preserve"><value>Delete</value></data>
+  <data name="EditLabelsDialogCancel" xml:space="preserve"><value>Cancel</value></data>
+  <data name="EditLabelsDialogSave" xml:space="preserve"><value>Save</value></data>
+  <data name="EditLabelsDialogDiscard" xml:space="preserve"><value>Discard</value></data>
+  <data name="MainChangesetWarningInstallingHidden" xml:space="preserve"><value>WARNING: INSTALLING HIDDEN MODULE ({0})</value></data>
```

Comment:
It's *only* displayed for AlertOnInstall; this is the warning string for AlertOnInstall. I'm not sure I understand what this comment is about.

Reply:
Indeed, I just saw that it is only displayed when `AlertOnInstall` is ticked. I though it would also be displayed for hidden modules.

However, that's where the problem is, the text warns about _"Installing a hidden module"_. but a mod with `AlertOnInstall` is not necessarily hidden.

Reply:
*Ohhhhhhhhhhhh* I get it now. Thanks! This originally came up in the context of "what to do if a hidden module is being installed", but you're right, the settings are independent. I need to think about the wording.

What about:

> WARNING: MODULE IN LABEL {0} BEING INSTALLED ({1})

Hopefully the user named the label something that would remind them of why they wanted the alert.

Reply:
You know English isn't my first language, so correct me if I'm wrong, but wouldn't you normally say:
> WARNING: MODULE WITH LABEL {0} BEING INSTALLED ({1})

Reply:
Yes, that makes more sense. I've been thinking of a label as an object with modules inside of it, due to how the data structure is implemented, but a user would definitely think of it as something that the module has.

Reply:
Latest commit changes it to:

> WARNING: INSTALLING MODULE WITH LABEL {0} ({1})

Reply:
Actually, I think the resource name should also be updated :grin: `MainChangesetWarningInstallingHidden` -> `MainChangesetInstallingModuleWithWarningTag` or something better you can come up with

Reply:
How about `MainChangesetWarningInstallingModuleWithLabel`?

Reply:
Yep, that's fine.